DataFormats/BTauReco/interface/PixelClusterTagInfo.h

```diff
@@ -0,0 +1,60 @@
+#ifndef DataFormats_BTauReco_PixelClusterTagInfo_h
+#define DataFormats_BTauReco_PixelClusterTagInfo_h
+
+#include "DataFormats/BTauReco/interface/BaseTagInfo.h"
+
+namespace reco {
+
+  struct PixelClusterProperties {
+    float x = 0;
+    float y = 0;
+    float z = 0;
+    int charge = 0;
+    int layer = 0;
+  };
+
+  struct PixelClusterData {
+    int8_t R004[4] = {};
+    int8_t R006[4] = {};
+    int8_t R008[4] = {};
+    int8_t R010[4] = {};
+    int8_t R016[4] = {};
+    int8_t RVAR[4] = {};
+    unsigned int RVWT[4] = {};
```

Comment:
CMS code rules require lowercase initials for class member variables: "r004", ..., "rVWT" (or "rvwt", or whatever you think it is most appropriate)